src/modules/LanUdevAuto.rb

```diff
@@ -205,7 +207,7 @@ def Write
       end
 
       if !rules.empty? && AllowUdevModify()
-        SetAllLinksDown()
+        SetAllLinksDown() if !Linuxrc.usessh
```

Comment:
but if release managers are fine with it, then at least please add TODO comment capturing your findings